source/Octopus.Tentacle.Kubernetes.Tests.Integration/Setup/KubernetesAgentInstaller.cs

```diff
@@ -150,8 +150,9 @@ string BuildAgentInstallArguments(string valuesFilePath, string? tentacleImageAn
 
     async Task<string> GetAgentThumbprint()
     {
-        string? thumbprint = null;
+        var maxAttempts = 10;
```

Comment:
Honestly I'd be happy to bump this to 60 and see what happens. A minute isn't a long time in this test suite